src/Client/TrapHandle/ContextProvider/Source.php

```diff
@@ -17,8 +16,8 @@
  * @author Nicolas Grekas <p@tchwork.com>
  * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
  *
- * @link https://github.com/symfony/var-dumper/blob/7.0/Dumper/ContextProvider/SourceContextProvider.php
- * @link https://github.com/symfony/var-dumper/blob/6.3/Dumper/ContextProvider/SourceContextProvider.php
+ * @see https://github.com/symfony/var-dumper/blob/7.0/Dumper/ContextProvider/SourceContextProvider.php
+ * @see https://github.com/symfony/var-dumper/blob/6.3/Dumper/ContextProvider/SourceContextProvider.php
```

Comment:
yes, `@see` may contain a web page link but we use `@link` for that. It's semantically closer

Reply:
Added a rule for @link